packages/fxa-shared/guards/AdminPanelGuard.ts

```diff
@@ -15,11 +21,14 @@ export enum PermissionLevel {
 export enum AdminPanelFeature {
   AccountLogs = 'AccountLogs',
   AccountSearch = 'AccountSearch',
-  ConnectedServices = 'ActiveSessions',
+  AccountHistory = 'AccountHistory',
+  ConnectedServices = 'ConnectedServices',
+  LinkedAccounts = 'LinkedAccounts',
   ClearEmailBounces = 'ClearEmailBounces',
-  DisableAccounts = 'DisableAccounts',
+  DisableAccount = 'DisableAccount',
```

Comment:
Q: Any reason this went from plural to singular?
I don't have any strong moral objections, just noticed that the snippet below's `name` was still pluraled:

```js
  [AdminPanelFeature.DisableAccount]: {
    name: 'Disable Accounts',
    level: PermissionLevel.Admin,
  },
```

Reply:
I think the change from plural to singular makes sense thinking about what the admin/support agent can do per each account (view their connected services, view their linked accounts, disable their account), but yeah I don't have a strong preference - agreed the `name`s could use a tweak to match.

Reply:
I prefer the singular. I've updated the name field accodingly.